release_notes_generator/utils/utils.py

```diff
@@ -53,3 +57,12 @@ def get_change_url(
         changelog_url = f"https://github.com/{repo.full_name}/compare/{rls.tag_name}...{tag_name}"
 
     return changelog_url
+
+
+def detect_row_format_invalid_keywords(row_format: str, row_type: str = "Issue") -> list[str]:
```

Comment:
You do miss a method docstring here.

Reply:
Addressed in commit aee3c5f4428a5d0b27b8f44dad6241a36ea8f7d6.